server/src/test/java/de/uftos/services/TimetableServiceTests.java

```diff
@@ -0,0 +1,78 @@
+package de.uftos.services;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+
+import de.uftos.entities.Break;
+import de.uftos.entities.Lesson;
+import de.uftos.entities.Server;
+import de.uftos.entities.Timetable;
+import de.uftos.entities.TimetableMetadata;
+import de.uftos.repositories.database.ServerRepository;
+import de.uftos.repositories.database.TimetableRepository;
+import de.uftos.repositories.ucdl.UcdlRepository;
+import de.uftos.repositories.ucdl.parser.javacc.ParseException;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+import org.springframework.web.server.ResponseStatusException;
+
+@SuppressWarnings("checkstyle:MissingJavadocType")
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+public class TimetableServiceTests {
+
+  @Mock
+  private TimetableRepository timetableRepository;
+  @Mock
+  private UcdlRepository ucdlRepository;
+
+  @Mock
+  private ServerRepository serverRepository;
```

Comment:
Are the server and ucdl repositories needed anywhere?

Reply:
You are right, they are no longer used. I removed them